assets/src/stories-editor/components/index.js

```diff
@@ -24,6 +24,8 @@ export { default as withHasSelectedInnerBlock } from './higher-order/with-has-se
 export { default as withPageNumber } from './higher-order/with-page-number';
 export { default as withStoryFeaturedImageNotice } from './higher-order/with-story-featured-image-notice';
 export { default as withEditFeaturedImage } from './with-edit-featured-image';
+export { default as withVideoBlockPreview } from './with-video-block-preview';
+export { default as VideoBlockEditWithPreview } from './video-block-edit-with-preview';
```

Comment:
Given that it does more than just enabling autoplay/previews for the video, I suggest renaming `VideoBlockEditWithPreview` to `CustomVideoBlockEdit` and `withVideoBlockPreview` to `withCustomVideoBlockEdit` to make it more clear what it does.